packages/backend/src/panels/YeomanUIPanel.ts

```diff
@@ -37,7 +38,23 @@ export class YeomanUIPanel extends AbstractWebviewPanel {
     }
   }
 
-  public async loadWebviewPanel(uiOptions?: any) {
+  private setYoUiPromises() {
+    this.yoUiCommandPromise = new Promise((resolve: ResolveType, reject: RejectType) => {
+      this.yoUiFlowPromise = { resolve, reject };
```

Comment:
why is `yoUiFlowPromise` called a "promise"?
It does not implement the Promise interface.
It is maybe a "Deferred Promise"

e.g `yoUiCommandPromise` has a `.then` method.
but `yoUiFlowPromise` does not

Reply:
Done